packages/kernels/mocaccino-lts/sources/definition.yaml

```diff
@@ -1,8 +1,7 @@
 name: mocaccino-lts-sources
 category: kernel
-version: "5.4.86"
+version: "5.4.86+1"
 prefix: linux
```

Comment:
You seems to not use the `prefix` which was declared here?

Reply:
About `prefix` do you mean why is yet there or what? I'm not sure to drop `prefix` because the same field is used in autobump hooks. Is used there or it's only a coincidence of the naming. I don't investigate in the autobump code.